ansi2html/converter.py

```diff
@@ -347,9 +373,36 @@ def _vt100_box_drawing():
             yield ansi[last_end:]
 
         ansi = "".join(_vt100_box_drawing())
+        def _osc_link(ansi):
+            last_end = 0
+            for match in self.osc_link_re.finditer(ansi):
+                trailer = ansi[last_end : match.start()]
+                yield trailer
+                url = match.groups()[0]
+                text = match.groups()[1]
+                yield OSC_Link(url, text)
+                last_end = match.end()
+            yield ansi[last_end:]
 
         state = _State()
-        inside_span = False
+        for part in _osc_link(ansi):
+            if isinstance(part, OSC_Link):
+                yield part
+            else:
+                if ((sys.version_info.major == 3 and sys.version_info.minor >= 3)
+                        or sys.version_info.major > 3):
```

Comment:
This makes no sense because we already require py36+, please remove variant for relic-pythons.

Reply:
Thanks for the comment, I have removed the check in a new commit.